cli/crates/server/src/servers.rs

```diff
@@ -104,18 +104,25 @@ async fn server_loop(
             result = spawn_servers(worker_port, bridge_port, watch, sender.clone(), event_bus.clone(), path_changed.as_deref(), tracing) => {
                 result?;
             }
-            path = wait_for_event_and_match(receiver, |event| match event {
-                Event::Reload(path) => Some(path),
-                Event::BridgeReady => None,
+            event = wait_for_event_and_match(receiver, |event| match event {
```

Comment:
Doesn't seem really useful as a change here

Reply:
Right, it was needed for a more substantial change that has since been removed from this PR. Reverting this too.